certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
 ## 4.1 Certificate Application
 
 ### 4.1.1 Who can submit a certificate application
-In accordance with Section 5.5.2, the CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests.
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CA SHALL use this information to identify subsequent suspicious certificate requests.
+
+For the Root and Subordinate CAs:
+- An application for a CA certificate shall be submitted by an authorized representative of the applicant CA.
+
+For end entity certificates:
+- A certificate application shall be submitted to the CA by the Subscriber, an authorized organization representative, or an RA on behalf of the Subscriber. 
```

Comment:
In that case, for an issuing CA on the civilian side to issue DV certificates to anyone demonstrating technical control, then that issuing CA would need to define "authorized representative" as "any applicant capable of performing successful domain validation as defined in section 3.2.2.4". This is sort of a no-op, and the same as saying there is no level of authorization required beyond what's already in the BRs.

@lachellel Is that what you would recommend and see as desired for a civilian issuing CA?

It seems to me that we'd be better off with a CP that leaves more wiggle room, and says that issuing CAs _may_ choose to define additional practices to validate that applicants for end entity certificates are authorized to do so.